spec/support/date_formatter_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
```

Comment:
I'm including the rails test helper here, since I need Rails (see above where I'm including a module thats a part of Rails). Objects that don't use Rails at all should instead use `require 'spec_helper'`, which is waay faster since it doesn't load a ton of stuff.